<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="row_19"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_0"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_3"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_4"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_5"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_6"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_7"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_8"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_9"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_10"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_11"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_12"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_13"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_14"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_15"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_16"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_17"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_18"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
